### PR TITLE
Updates to support acoustic datasets

### PR DESCRIPTION
**Pull request by jcermauwedu**
* Update the URL to point to the current documentation
 * Allow `options` to pass from compliance checker into the plugin.
 * Add option to skip attribute check on variables for ancillary_variables
 * Migrate to f-string formatting as suggested by ruff

### CODE REVIEW

File: cc_plugin_glider/glider_dac.py [hunk 1]
```diff
@@ -26,12 +26,14 @@ class GliderCheck(BaseNCCheck):
     _cc_spec_version = "3.0"
     _cc_checker_version = __version__
     _cc_url = (
-        "https://github.com/ioos/ioosngdac/wiki/NGDAC-NetCDF-File-Format-Version-2"
+        "https://ioos.github.io/glider-dac/ngdac-netcdf-file-format-version-2"
```
Comment (ocefpaf) [inline cc_plugin_glider/glider_dac.py:29]: 👍🏽

**Comment by ocefpaf** [comment]
> * Migrate to f-string formatting as suggested by ruff

@jcermauwedu I'm all for fixing these style issue but when adding new code let's avoid mixing the changes b/c it makes it a bit harder to follow the syntax changes vs the actual code changes. No need to revert them here but this is something we should try do in the future.

File: cc_plugin_glider/util.py [hunk 2]
```diff
@@ -73,6 +73,9 @@ def _check_variable_attrs(dataset, var_name, required_attributes=None):
 
     # Get the expected attrs to check
     check_attrs = required_attributes or required_var_attrs.get(var_name, {})
+    if not options is None:
```
Comment (ocefpaf) [inline cc_plugin_glider/util.py:76]: Just a suggestion but I think this reads better. 

```suggestion
    if options is not None:
```

File: cc_plugin_glider/util.py [hunk 3]
```diff
@@ -57,7 +57,7 @@ def _check_dtype(dataset, var_name):
     return (score, out_of, messages)
 
 
-def _check_variable_attrs(dataset, var_name, required_attributes=None):
+def _check_variable_attrs(dataset, var_name, required_attributes=None, options=None):
```
Comment (ocefpaf) [inline cc_plugin_glider/util.py:60]: Let's document this new kw in the docstring below. In my mind I was think that this added options for the function but it is adding variables that the checks are optional, right? Or maybe variables that you want to skip? If the latter, maybe let's rename this kw to "skip_variables" and document that the user should pass a list of variables to skip. That way we can drop the complex "no_var-name" check below and just drop whatever is in the list.

Does that make sense?

File: cc_plugin_glider/glider_dac.py [hunk 4]
```diff
@@ -752,8 +746,7 @@ def check_valid_min_dtype(self, dataset):
             if valid_min is not None:
                 test_ctx.assert_true(
                     util.compare_dtype(np.dtype(valid_min_dtype), ncvar.dtype),
-                    "{}:valid_min has a different data type, {}, than variable {}, "
-                    "{}".format(var_name, valid_min_dtype, var_name, str(ncvar.dtype)),
+                    f"{var_name}:valid_min has a different data type, {valid_min_dtype}, than variable {var_name}, {str(ncvar.dtype)}"
```
Comment (ocefpaf) [inline cc_plugin_glider/glider_dac.py:749]: The f-string already stringfy it. No need to call `str` on it.

**Comment by jcermauwedu** [comment]
Refactoring required for option handling.

**Pull request closed without merging**